gtest_parallel.py

```diff
@@ -185,6 +184,11 @@ def _normalize(string):
 
   @staticmethod
   def _logname(output_dir, test_binary, test_name, execution_number):
+    if output_dir is None:
+      (log_handle, log_name) = tempfile.mkstemp(prefix='tmpGP_', suffix=".log")
```

Comment:
Would it be reasonable to prefix with `tmp_gtest_parallel_`, or just `gtest_parallel_` or is that too long? `tmpGP` doesn't really read as `gtest-parallel` if you don't know where the file comes from to begin with. Omitting `tmpGP_` is also fine, but I like the prefix idea.

Reply:
changed to `gtest_parallel_`